cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -190,8 +190,15 @@
     {# Check and include template-level JavaScript. #}
     {% include '/js/routes/' + request.path.split('/')[1] + '/single.js' ignore missing %}
 
+    {# TODO: Remove conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% include '/js/routes/' + request.path[1:] + 'index.js' ignore missing %}
+    {% if page %}
+        {% for js in page.media %}
```

Comment:
Why are we writing out multiple scripts instead of integrating with Webpack? I'm not saying we have to use https://github.com/markfinger/python-webpack but it may provide some insight.